app/templates/minor/profile.html

```diff
@@ -1,92 +1,561 @@
+        
+                    <div class="form-group">
+                        <label><strong>Which of the following existing community-based experiences will you take part in during the summer? Please select one.</strong></label><br>
+                        <input type="radio" id="internship" name="experienceType" value="Internship" {% if summer_experience.experienceType == 'Internship' %}checked{% endif %} required>
+                        <label for="internship">Internship</label><br>
+                        <input type="radio" id="cbr" name="experienceType" value="Community-based research" {% if summer_experience.experienceType == 'Community-based research' %}checked{% endif %}>
+                        <label for="cbr">Community-based research (CBR) within URCPP or other research that meets established criteria</label><br>
+                        <input type="radio" id="communityLabor" name="experienceType" value="Community-Based Labor Position" {% if summer_experience.experienceType == 'Community-Based Labor Position' %}checked{% endif %}>
+                        <label for="communityLabor">Community-Based Labor Position, either in Berea or through the Off-Campus Summer Labor Experience</label><br>
+                        <input type="radio" id="epgInternship" name="experienceType" value="EPG Summer Internship" {% if summer_experience.experienceType == 'EPG Summer Internship' %}checked{% endif %}>
+                        <label for="epgInternship">EPG Summer Internship</label><br>
+                        <input type="radio" id="bonnerService" name="experienceType" value="Bonner Scholar Summer of Service" {% if summer_experience.experienceType == 'Bonner Scholar Summer of Service' %}checked{% endif %}>
+                        <label for="bonnerService">Bonner Scholar Summer of Service</label>
+                        <br>
+                        <input type="radio" id="otherExperience" name="experienceType" value="Other" {% if summer_experience.experienceType not in ['Internship', 'Community-based research', 'Community-Based Labor Position', 'EPG Summer Internship', 'Bonner Scholar Summer of Service'] %}checked{% endif %} required>
+                        <label for="otherExperience">Other. Briefly describe your summer experience below if it will not be one of the other experiences listed.</label><br>
+                        <textarea class="form-control" rows="2" cols="72" id="otherExperienceDescription" name="otherExperienceDescription" {% if summer_experience.experienceType in ['Internship', 'Community-based research', 'Community-Based Labor Position', 'EPG Summer Internship', 'Bonner Scholar Summer of Service'] %}style="display:none;"{% endif %}>{{ summer_experience.experienceType if summer_experience.experienceType not in ['Internship', 'Community-based research', 'Community-Based Labor Position', 'EPG Summer Internship', 'Bonner Scholar Summer of Service'] else '' }}</textarea><br>
+                    </div>
```

Comment:
We shouldn't do that `in` with a hard-coded list of text 3 different times. Figure it out once and store that in a boolean.